internal/manifests/console/consolelink.go

```diff
@@ -71,6 +73,22 @@ func CreateOrUpdateConsoleLink(ctx context.Context, c client.Client, cl *console
 	return nil
 }
 
+func DeleteKibanaConsoleLink(ctx context.Context, c client.Client) error {
+
+	current := &consolev1.ConsoleLink{}
+	current.Name = KibanaConsoleLinkName
```

Comment:
Why this separate? You can merge them right like `current := &consolev1.ConsoleLink{Name: KibanaConsoleLinkName}`

Reply:
I've tried doing it like this before, but it gave a compiler error: `unknown field Name in struct literal`

Reply:
You can use the helper function in `build.go` like this
`current := NewConsoleLink(KibanaConsoleLinkName, "", "", "")`

Reply:
Right, how did I miss that..
I'll update the code